examples/upload-vmck-json.sh

```diff
@@ -0,0 +1,3 @@
+nomad job run -output vmck.nomad > vmck.json # convert hcl to json file for upload on server TODO: custom nomand and json filename
```

Comment:
Please rename the script to `cluster.sh`, it expresses the intention of the action, instead of how it's implemented.